aleth/main.cpp

```diff
@@ -566,6 +566,12 @@ int main(int argc, char** argv)
         {
             configPath = vm["config"].as<string>();
             configJSON = contentsString(configPath.string());
+
+            if (configJSON.empty())
+            {
+                cout << "Config file not found or empty (" << configPath.string() << ")\n";
```

Comment:
Output to `cerr` like other messages in this file

Reply:
@gumb0 Amended :)